.github/workflows/lumo.yml

```
@@ -33,3 +33,9 @@ jobs:
           SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
           SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
         run: yarn test:lumo
+
+      - uses: actions/upload-artifact@v2
+        if: failure()
```

According to the documentation, should look like this:

```suggestion
        if: ${{ failure() }}
```